### ISSUE TITLE
Sync-batchnorm vs standard batchnorm

### ISSUE THREAD

**Issue by Cold-Winter** [AliaksandrSiarohin/first-order-model#423]
I reimplement your code in tensorflow and find that standard batchnorm can always get sharp images (visually look better) than sync batchnorm. Do you have the same observations?

I also find that the model is sensitive to initialization and data shuffling. I get very different results even if i use the same hyper-parameter to run different experiments.

**Comment by AliaksandrSiarohin** [comment]
I didn't run this code with regurar bn. I guess this could be the case.
